bare-metal/container-linux/kubernetes/profiles.tf

```diff
@@ -24,11 +45,11 @@ resource "matchbox_profile" "container-linux-install" {
 data "template_file" "container-linux-install-configs" {
   count = "${length(var.controller_names) + length(var.worker_names)}"
 
-  template = "${file("${path.module}/cl/container-linux-install.yaml.tmpl")}"
+  template = "${local.template_yaml}"
 
   vars {
-    container_linux_channel = "${var.container_linux_channel}"
-    container_linux_version = "${var.container_linux_version}"
+    os_channel              = "${local.channel}"
+    os_version              = "${var.os_version}"
     ignition_endpoint       = "${format("%s/ignition", var.matchbox_http_endpoint)}"
     install_disk            = "${var.install_disk}"
     container_linux_oem     = "${var.container_linux_oem}"
```

Comment:
Vaguely unrelated to your PR, but this will probably be dropped at some point. So don't worry about the oem setting.